SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -12,6 +10,25 @@ DEFINE_LOG_CATEGORY(LogSpatialEventTracer);
 
 namespace SpatialGDK
 {
+TraceQueryPtr ParseOrDefault(const FString& Str, const char* FilterForLog)
+{
+	TraceQueryPtr Ptr;
+	if (Str.Len() > 0)
+	{
+		Ptr.Reset(Trace_ParseSimpleQuery(TCHAR_TO_ANSI(*Str)));
+		UE_LOG(LogSpatialEventTracer, Log, TEXT("Applied %s query: %s"), FilterForLog, *Str);
+	}
+
+	if (!Ptr.IsValid())
+	{
+		UE_LOG(LogSpatialEventTracer, Warning, TEXT("The specified %s is invalid, sampling will be disabled. %s"), FilterForLog,
```

Comment:
[nit] `The specified query \"%s\" is invalid; defaulting to \"false\" query. %s`?

The main thing I want to avoid is using "sampling" terminology (since it's filtering which is similar but conceptually distinct).